### PR TITLE
currentsize added (issue #23)

### PR DESCRIPTION
**Pull request by chrthomsen**
Property added to handle issue #23. Is there a better name than currentsize?

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,10 @@ Version 2.7 (unreleased)
   to set an end date for the most current version without adding a new
   version.
 
+  A (read-only) property ``currentsize`` to ``BatchFactTable`` and
+  ``_BaseBulkloadable`` to get the number of inserted rows awaiting
+  to be loaded into the database table. (GitHub issue #23)
+
```
Comment (skejserjensen) [inline CHANGELOG.rst:24]: There seems to be an Added missing somewhere in this sentence.

File: pygrametl/tables.py [hunk 2]
```diff
@@ -1898,6 +1898,11 @@ def __insertexecutemany(self):
             self.targetconnection.executemany(self.insertsql, self.__batch)
             self.__batch = []
 
+    @property
+    def currentsize(self):
```
Comment (skejserjensen) [inline pygrametl/tables.py:1902]: Could `currentsize` be misunderstood as the size of the table in the database? Maybe `currentbatchsize` could be used, but then the API differs between batch loading and bulk loading Dimensions/FactTables.

File: pygrametl/tables.py [hunk 3]
```diff
@@ -2123,6 +2128,11 @@ def __preparetempfile(self):
         self.__filename = self.__namedtempfile.name
         self.__ready = True
 
+    @property
+    def currentsize(self):
```
Comment (skejserjensen) [inline pygrametl/tables.py:2132]: Could `currentsize` be misunderstood as the size of the table in the database? Maybe `currentbulksize` could be used, but then the API differs between batch loading and bulk loading Dimensions/FactTables.

**Review by skejserjensen — COMMENTED** [review]
Another name than `currentsize` would approve the API, but I do not have a good idea for a name that can be used for both methods.

File: CHANGELOG.rst (hunk 1 above)
Comment (chrthomsen) [reply CHANGELOG.rst:24]: Yes, that was to be consistent with the item above for closecurrent  (and they are both under the Added heading). But we could write "added" for both if we prefer that.

File: pygrametl/tables.py (hunk 2 above)
Comment (chrthomsen) [reply pygrametl/tables.py:1902]: Yes, I think we should use the same name in both places. I also thought about "awaiting". Is that easier to understand?

File: CHANGELOG.rst (hunk 1 above)
Comment (skejserjensen) [reply CHANGELOG.rst]: Writing as if the heading is the first word is fine, although we should probably do so consistently for the entire changelog. For example, at the moment we write added under **Version 2.2**.

File: pygrametl/tables.py (hunk 3 above)
Comment (qianxuanyon) [reply pygrametl/tables.py]: Can be understood as the number of rows currently affected
https://www.python.org/dev/peps/pep-0249/#rowcount

**Pull request merged**